x-pack/plugins/cloud_integrations/cloud_links/public/index.ts

```diff
@@ -4,7 +4,7 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-
+export type { CloudLinksDepsStart } from './plugin';
```

Comment:
I don't see where this is getting imported from

Reply:
That's my bad thanks - I had a context provider in a previous implementation where I needed this and I forgot to remove :face_palm: